src/main/resources/io/jenkins/plugins/ml/ResultAction/index.jelly

```diff
@@ -55,6 +55,7 @@
                             </j:if>
                             <li>
                                 <a href="${it.href}/${a.task}/${a.fileName}">${a.fileName}</a>
+                                <IMG SRC="${it.href}/${a.task}/${a.fileName}"/>
```

Comment:
Would it work if the image generated was very large?

It might be useful to also document that if the checked out project from Git contains images (I think) these images will be displayed in the action as well?

We probably want to display only the images that were generated with the plug-in in our action.

Reply:
I am not sure about large files. will test some and give a commit that ensures images only from a notebook.

Reply:
When I said large, it's not necessarily about the file size, more about the width x height. That might break the Jenkins UI and make it very slow. If so, maybe the links are the best option, or maybe a different approach (there's always a third, fourth, etc, option :-) )